src/utils/UndoableActionsHandlerProvider.js

```diff
@@ -57,7 +56,7 @@ const BaseActionNotification = ({ action, removeAction, closeSnackbar, addSnackb
             },
             contentOptions: {
                 actionText: "Undo",
-                actionHandler: handleCancel(action.id),
+                actionHandler: handleCancel,
```

Comment:
The comment does not apply to this line but I cannot select the correct one...

This useEffect has dependencies that are no longer used, erase them all, and let eslint fix it for you automatically, so that you only depend on what you need